src/ensemble/random_forest_classifier.rs

```diff
@@ -553,6 +554,37 @@ impl<T: RealNumber> RandomForestClassifier<T> {
         which_max(&result)
     }
 
+    /// Predict the per-class probabilties for each observation.
+    /// The probability is calculated as the fraction of trees that predicted a given class
+    pub fn predict_probs<M: Matrix<T>>(&self, x: &M) -> Result<DenseMatrix<f64>, Failed> {
```

Comment:
in scikit it is called predict_proba, I think that it is better to keep the same name